pycroft/lib/finance/retransfer.py

```diff
@@ -0,0 +1,47 @@
+from collections.abc import Sequence
+from datetime import datetime, timedelta
+
+from sepaxml import SepaTransfer
+from sqlalchemy import select
+from sqlalchemy.orm import joinedload
+
+from pycroft import config
+from pycroft.helpers.utc import ensure_tz
+from pycroft.model import session
+from pycroft.model.finance import BankAccountActivity
+
+
+def get_activities_to_return() -> Sequence[BankAccountActivity]:
+    statement = (
+        select(BankAccountActivity)
+        .options(joinedload(BankAccountActivity.bank_account))
+        .filter(BankAccountActivity.transaction_id.is_(None))
+        .filter(BankAccountActivity.amount > 0)
+        .filter(BankAccountActivity.imported_at < ensure_tz(datetime.utcnow() - timedelta(days=14)))
+    )
+
+    return session.session.scalars(statement).all()
```

Comment:
Also, please explicitly pass the `session` as per [ADR004](https://agdsn.github.io/pycroft/arch/adr-004.html):
```suggestion
def get_activities_to_return(session: Session) -> Sequence[BankAccountActivity]:
    statement = (
        select(BankAccountActivity)
        .options(joinedload(BankAccountActivity.bank_account))
        .filter(BankAccountActivity.transaction_id.is_(None))
        .filter(BankAccountActivity.amount > 0)
        .filter(BankAccountActivity.imported_at < ensure_tz(datetime.utcnow() - timedelta(days=14)))
    )

    return session.scalars(statement).all()
```